manifests/lens.pp

```diff
@@ -23,12 +28,12 @@
 #   }
 #
 define augeas::lens (
-  $ensure       = present,
-  $lens_content = undef,
-  $lens_source  = undef,
-  $test_content = undef,
-  $test_source  = undef,
-  $stock_since  = false,
+  String $ensure                 = present,
+  Optional[String] $lens_content = undef,
+  Optional[String] $lens_source  = undef,
+  Optional[String] $test_content = undef,
+  Optional[String] $test_source  = undef,
+  Optional[String] $stock_since  = undef,
```

Comment:
I don't think we accept empty strings for any of those parameters? In that case please update it to `String[1]`